src/Modbus.Common/Util/Extensions.cs

```diff
@@ -69,8 +69,9 @@ public static ushort GetUInt16(this ModbusObject register)
 		/// </summary>
 		/// <param name="list">The list of registers (min. 2).</param>
 		/// <param name="startIndex">The start index. Default: 0.</param>
+		/// <param name="bigEndianRegistryOrder">Change the order of WORDs in multi-registry data. True indicates Big Endian encoding, False Little Endian encoding. Default: True</param>
 		/// <returns></returns>
-		public static uint GetUInt32(this IEnumerable<ModbusObject> list, int startIndex = 0)
+		public static uint GetUInt32(this IEnumerable<ModbusObject> list, int startIndex = 0, bool bigEndianRegistryOrder = true)

```

Comment:
```suggestion
		public static uint GetUInt32(this IEnumerable<ModbusObject> list, int startIndex = 0, bool inverseRegisters = false)

```

It also takes credits of a good code style to use default values whereever possible.